esmvaltool/diag_scripts/ocean/diagnostic_tools.py

```diff
@@ -125,7 +125,10 @@ def bgc_units(cube, name):
     if name in ['tos', 'thetao']:
         new_units = 'celsius'
 
-    if name in ['no3', ]:
+    if name in ['sos', 'so']:
+       cube.units = '0.001'
```

Comment:
Identation not correct at the line 129. The tests should pass after fixing this.

Reply:
ok